Meta/Lagom/Tools/CodeGenerators/LibWeb/WrapperGenerator/IDLGenerators.cpp

```diff
@@ -1345,6 +1351,18 @@ static void generate_wrap_statement(SourceGenerator& generator, String const& va
     } else if (type.name == "unsigned long") {
         scoped_generator.append(R"~~~(
     @result_expression@ JS::Value((u32)@value@);
+)~~~");
+    } else if (type.name == "long long") {
+        scoped_generator.append(R"~~~(
+    auto bigval = ::Crypto::SignedBigInteger::create_from((i64)@value@);
+    
+    @result_expression@ JS::Value(bigval.to_double());
```

Comment:
This seems pointless... just cast the value to a double directly?

Reply:
Youre right. The `choosing the numeric value with an even significand if there are two equally close values` had me a bit confused, but it seems that its the default rounding mode for IEEE 754, and thus nothing to worry about.